examples/input_specification.txt

```diff
@@ -0,0 +1,22 @@
+output_image_path

```

Comment:
A future improvement could be implement reading standard 3D files (like `.obj`) so that objects can be modeled in a program like 3DS MAX and exported.